src/routes/+page.svelte

```diff
@@ -8,10 +9,15 @@
 
 <div>
     <h1 class="text-3xl font-bold underline">Hello world!</h1>
+    <TextField
+        textFieldText="Enter Recipient Address or EOS Account"
+        errorCriteria={(val) => /^0x[a-fA-F0-9]{40}$/.test(val)}
```

Comment:
Blocker: This will turn into an error state if someone puts in their EOS account name (e.g.  `ericpassmore`).

Reply:
So in the context of the faucet, I think it should go into an error state for an account name because the faucet is expecting a wallet address. But that limits its use to just the faucet so we may need a way to specify what error criteria is to be used. Idk if that's something we should handle via a prop that specifies what to check for. Or if we let the check be a prop the client passes in and then each client could specify what check to run